vhost-device-sound/CHANGELOG.md

```diff
@@ -9,6 +9,20 @@
 
 ### Deprecated
 
+## v0.1.1
+
+### Added
+- pipewire: specify audio channel position
+
+### Changed
+- Update pipewire dependencies to version 0.8 after the release of pipewire
+  v0.8
+
+### Fixed
+- Fix symbolic links to license files
+- Remove duplicate increment in pipewire
+- Destroy pipewire streams not destroyed in pipewire
+
```

Comment:
What about adding links to the PRs, something like we do in vhost: https://raw.githubusercontent.com/rust-vmm/vhost/main/vhost/CHANGELOG.md